### PR TITLE
IBX-7873: Add links to interactive demos to dev guide

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | IBX-7873 |
| Versions      | 4.6 and up |
| Edition       | all |

Add links to interactive demos to dev guide

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -331,6 +331,7 @@ nav:
         - Add remote PIM support: pim/add_remote_pim_support.md
     - Commerce:
         - Commerce: commerce/commerce.md
+        - Interactive demo test: commerce/interactive_demo.md
```
Comment (mnocon) [inline mkdocs.yml:334]: Embedding the full demo doesn't work for me, it covers navigation and breaks the page. I suppose we should remove this entry and the test file?

**Review by mnocon — COMMENTED** [review]
Thanks, I like the result!

File: mkdocs.yml (hunk 1 above)
Comment (mnocon) [reply mkdocs.yml:334]: Yep, I see that now, I didn't know it when writing this comment 😄

**Comment by juskora** [comment]
General comment, I think we can discuss about a separate space for all those videos 'cause in this form it looks little bit messy as these video are placed in different places, hard to find at first glance and lacks consistency.

**Comment by julitafalcondusza** [comment]
I agree with @juskora - it lacks consistency. Each page should have same placement of the video. 
In "Content management product guide" you can easily miss interactive demo - it is placed in the middle of the content, between many other images, so you just keep scrolling. 
I like the idea of "Click to see an interactive demo" frame. I think that you could use it in all the pages.
So: same placement, same style.

**Pull request closed without merging**